Implementing Shor's algorithm using the Qrisp framework on Cat Qubits.ipynb

```diff
@@ -0,0 +1,770 @@
+{
```

Comment:
Can you rename the notebook to something like 5 - Shor's algorithm on logical qubits to be aligned with the other notebooks name? (thus, the one on quantum teleportation would become num 6)

Reply:
Indeed: "Shor's algorithm on logical qubit using the Qrisp framework"?

Reply:
qubitS with an S and yes perfect for me